tests/resources/test_resource_links.py

```diff
@@ -64,3 +60,18 @@ def test_update_links(app, client, input_data, headers):
 
     assert res.status_code == 200
     assert_expected_links(pid_value, res.json["links"])
+
+
+def test_config_hostname_links(app, client, input_data, headers):
+    site_hostname = "testsite.com"
+    app.config['SITE_HOSTNAME'] = site_hostname
+
+    res = client.post('/mocks', headers=headers, json=input_data)
+    pid_value = res.json["id"]
+    data = res.json
+    data['metadata']['title'] = 'New title'
+
+    res = client.get(f'/mocks/{pid_value}', headers=headers, json=data)
```

Comment:
We can remove this for the purpose of this test. The API response to POST should include links that have the app's SITE_HOSTNAME and can be used in the assertion directly.

Reply:
Indeed, thanks!